superset/views/core.py

```diff
@@ -1056,13 +1056,14 @@ def get_query_string_response(self, viz_obj):
 
     @log_this
     @has_access_api
-    @expose('/explore_json/<datasource_type>/<datasource_id>/')
+    @expose('/explore_json/<datasource_type>/<datasource_id>/', methods=['POST'])
```

Comment:
While you are in here... I would love to get rid of the trailing `/` everywhere
basically add another line
`@expose('/explore_json/<datasource_type>/<datasource_id>', methods=['POST'])`